releasenotes/notes/Deprecate_legacy_pulse_builder_command-2a3c8dcbceae624e.yaml

```diff
@@ -0,0 +1,63 @@
+---
+deprecations:
+  - |
+    Deprecated injecting circuit gate operation into the pulse context.
+    
+    Deprecated
+    :func:`~qiskit.pulse.builder.call_gate`,
+    :func:`~qiskit.pulse.builder.cx`,
+    :func:`~qiskit.pulse.builder.u1`,
+    :func:`~qiskit.pulse.builder.u2`,
+    :func:`~qiskit.pulse.builder.u3`,
+    :func:`~qiskit.pulse.builder.x`
+
+    .. code-block:: python
+
+      """ Users are encouraged to use gates only in relm of circuits, NOT in pulse(s)
+        Pulse builder should exclusively be used with pulse schedules (like Drag, Gaussian, GaussianSqaure, etc.)
+        Example:"""
+      with pulse.build() as only_pulse_scheds:
+        pulse.play(Drag(180, 0.2, 40, -1.1), DriveChannel(0))
+        pulse.play(Gaussian(160, 0.14, 30), DriveChannel(1))
+        
+      """If a user wishes to get the pulse schedules for a gate.
+      This example shows how to get pulse gate instructions."""
+      from qiskit.providers.fake_provider import FakePerth
+      backend=FakePerth()
+      sched=backend.target['x'][(qubit,)].calibration
+
+      # For BackendV1, like FakeNairobi
+      from qiskit.providers.fake_provider import FakeNairobi
+      FakeNairobi().defaults().instruction_schedule_map.get('x', (qubit,))
+
+      """If a user wishes to get the pulse schedules of a QuantumCirucit"""
+      from math import pi
+      qc = QuatumCircuit(2)
+      qc.rz(pi/2, 0)
+      qc.sx(0)
+      qc.rz(pi/2, 0)
+      qc.cx(0, 1)
+      
+      from compiler import schedule, transpile
+      qc_t = transpile(qc, backend)
+      schedule(qc_t, backend)
+    
+    Deprecated
+    :func:`~qiskit.pulse.builder.active_transpiler_settings`
+    :func:`~qiskit.pulse.builder.active_circuit_scheduler_settings`
+    :func:`~qiskit.pulse.builder.transpiler_settings`
+    :func:`~qiskit.pulse.builder.circuit_scheduler_settings`
+
+    Modified module doc of :mod:`qiskit.pulse.builder` related with example
+    code with circuit elements.
+
+    Deprecated arguments :code:`default_transpiler_settings`, :code:`default_circuit_scheduler_settings`
+    in :func:`~qiskit.pulse.builder.build`
+
+    Deprecated argument of :class:`.QuantumCircuit` type in :func:`~qiskit.pulse.builder.call`
+    
+    Modified related tests in
+    :file:`test/python/pulse/test_builder.py`,
+    :file:`test/python/pulse/test_block.py`
+    :file:`test/python/pulse/test_builder_v2.py`
+    :file:`test/python/transpiler/test_calibrationbuilder.py`
```

Comment:
```suggestion
```
You don't need to write about test because release note is communication tool for end users.

Reply:
Applied suggestion :)